src/applications/login/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "appName": "Login Page",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "login-page",
+  "rootUrl": "/login",
+  "production": false,
+  "template": {
+    "vagovprod": false,
```

Comment:
Is this the only thing that needs to be added to make feature flag/restrict this to staging?